dashboard/modules/node/node_head.py

```diff
@@ -246,6 +247,35 @@ async def get_node_module_internal_state(self, req) -> aiohttp.web.Response:
         )
 
     async def get_nodes_logical_resources(self) -> dict:
+
+        from ray.autoscaler.v2.utils import is_autoscaler_v2
+
+        if is_autoscaler_v2():
+            from ray.autoscaler.v2.sdk import get_cluster_status
+
+            try:
+                cluster_status = get_cluster_status()
+            except Exception:
+                logger.exception("Error getting cluster status")
+                return {}
+
+            per_node_resources = {}
+            # TODO(rickyx): we should just return structure data rather than strings.
+            for node in cluster_status.healthy_nodes:
+                if not node.resource_usage:
+                    continue
+
+                usage_dict = {
+                    r.resource_name: (r.used, r.total)
+                    for r in node.resource_usage.usage
+                }
+                per_node_resources[node.node_id] = "\n".join(
+                    parse_usage(usage_dict, verbose=True)
+                )
+
+            return per_node_resources
```

Comment:
Do you have a example data for per_node_resources? I would like to confirm the frontend would render as expected

Reply:
```

{
    'nodeLogicalResources': {
            '6eb8d224b678a6793679cc189e7bc3ef787033005bc3de61ad3d29a2': '0.0/1.0 CPU\n0B/27.53GiB memory\n0B/13.76GiB object_store_memory'
    }
}
```

Reply:
also see the added e2e test.

Reply:
Thanks!